UnitsNet/GeneratedCode/Quantities/Density.NetFramework.g.cs

```diff
@@ -68,6 +68,7 @@ static Density()
                     new UnitInfo<DensityUnit>(DensityUnit.KilogramPerCubicCentimeter, BaseUnits.Undefined),
                     new UnitInfo<DensityUnit>(DensityUnit.KilogramPerCubicMeter, BaseUnits.Undefined),
                     new UnitInfo<DensityUnit>(DensityUnit.KilogramPerCubicMillimeter, BaseUnits.Undefined),
+                    new UnitInfo<DensityUnit>(DensityUnit.KilogramPerLiter, BaseUnits.Undefined),
```

Comment:
Was not regenerated after BaseUnits added

Reply:
Fixed.